assets/js/partials/site_scripts.js

```diff
@@ -1,20 +1,22 @@
 /**
  * Sticky header
+ * mispell as JS comment
  */
 
-jQuery(document).on("scroll", function() {
+jQuery(document).on("scroll", function () {
 	if (jQuery(document).scrollTop() > 0) {
 		jQuery("header, body").addClass("shrink");
 	} else {
 		jQuery("header, body").removeClass("shrink");
 	}
+	jQuery(".mispell").css("Display", "none");
 });
 
-jQuery(document).ready(function(jQuery) {
+jQuery(document).ready(function (jQuery) {
```

Comment:
🚫 **[phpcs]** <PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Opening parenthesis of a multi-line function call must be the last content on the line